claat/parser/md/README.md

```diff
@@ -108,9 +124,46 @@ This will appear in a negative info box.
 
 Codelabs sometimes contain links to SDKs or sample code. The codelab renderer
 will apply special button-esque styling to any link that begins with the word
-"Download".
```

Comment:
This is a little confusing to me - are we saying that either of these is an option? If so maybe we should make them 2 separate code blocks, like:

you can do this:
```
[Download thing]()
```
or this:
```
<button>
  [Download thing]()
</button>
```

And now that I type that, I'm a little more confused - doesn't the second example contain the first example? What's the difference?

Reply:
Good catch! I forgot to update the text which was wrong, currently the only way to render download buttons is [with a <button>](https://github.com/googlecodelabs/tools/pull/379#issuecomment-613709303), I will update the text to reflect that, or, alternatively, try make it work with regular links?